File: letter/ru/Д.py
from common import *

# TODO Почему то desmos оставляет пустым пересечение прямоугольников, поэтому пришлось их немного разделить
micro_eps = 0.01

expr = Eq(
    div(
        (Abs(x - y - 1 - micro_eps) + Abs(x + y + 1 + micro_eps) - 1)
        * (Abs(2 * x - S(2) / 3 * y + S(1) / 6) + Abs(2 * x + S(2) / 3 * y - S(1) / 6) - 1),
        sqrt(y + 1)
    )
    ,
    0
)
size = 1

File: letter/ru/Д.js
import { Letter } from '../../Letter.js';

const micro_eps = 0.01;
const expr = `(abs(x - y - 1 - ${micro_eps}) + abs(x + y + 1 + ${micro_eps}) - 1) * (abs(2*x - 2/3*y + 1/6) + abs(2*x + 2/3*y - 1/6) - 1) / sqrt(y + 1)`;

export const letter = new Letter(expr, 1);
